### PR TITLE
Update Marker test

### PR DESCRIPTION
**Pull request by iche033**
Signed-off-by: Ian Chen <ichen@osrfoundation.org>

# 🦟 Bug fix

## Summary

Added more tests to increase code coverage

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: src/Marker_TEST.cc [hunk 1]
```diff
@@ -118,12 +143,66 @@ void MarkerTest::Marker(const std::string &_renderEngine)
   rendering::unloadEngine(engine->Name());
 }
 
+/////////////////////////////////////////////////
+void MarkerTest::Material(const std::string &_renderEngine)
+{
+  if (_renderEngine == "optix")
+  {
+    igndbg << "Marker not supported yet in rendering engine: "
+            << _renderEngine << std::endl;
```
Comment (ahcorde) [inline src/Marker_TEST.cc:152]:
```suggestion
    igndbg << "Marker not supported yet in rendering engine: "
           << _renderEngine << std::endl;
```

File: src/Marker_TEST.cc [hunk 2]
```diff
@@ -118,12 +143,66 @@ void MarkerTest::Marker(const std::string &_renderEngine)
...
+    return;
+  }
+
```
Comment (ahcorde) [inline src/Marker_TEST.cc:155]:
```suggestion
```

File: src/Marker_TEST.cc (hunk 1 above)
Comment (iche033) [reply src/Marker_TEST.cc]: [c7dcb3b](https://github.com/gazebosim/gz-rendering/pull/673/commits/c7dcb3bb11895c7f53ea82807567867e29771954)

File: src/Marker_TEST.cc (hunk 2 above)
Comment (iche033) [reply src/Marker_TEST.cc:155]: [c7dcb3b](https://github.com/gazebosim/gz-rendering/pull/673/commits/c7dcb3bb11895c7f53ea82807567867e29771954)

**Review by chapulina — APPROVED** [review]

**Pull request merged**